### PR TITLE
Brazilian Portuguese

### PR DESCRIPTION
**Pull request by teuscard**
A copy of the App Store file in English has been created for the Brazilian Portuguese version. It was fully translated in the commit ad5a2a4 following the guidelines proposed.

This pull request will be created as draft because the file will still be revised one more time by me and the App file will still be created and translated. Therefore, this pull request only aims to inform that I am working on the Brazilian Portuguese version carefully. As soon as it is completed, this pull request will be marked as ready for review.

### CODE REVIEW

File: pt-BR App Store [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+Intuitivo - Relatórios visuais
+Preciso - Super Previsão
+Temas - Aparência Escura do iOS 13 + 16 outros
+Tempestades - Radar + Dark Sky
+Detalhado - 72 horas, 10 tipos de dados
+Inteligente - Travel Assist for home weather
+Inteligente - Assistente de Viagem para o tempo em casa
+Plano - com médias históricas
+Alerta - envie à família
+live, weather, map, forecast, radar, local, alert, rain, fall, winter, storm, check, track, tracker, hurricane" = "ao vivo, tempo, mapa, previsão, radar, local, alerta, chuva, outono, inverno, tempestade, verifique, rastreie, rastreador, furacão
```
Comment (teuscard) [inline pt-BR App Store:17]: Each word is repeated after the equal sign. Am I supposed to translate twice or leave one side in English?

File: pt-BR App Store [hunk 2]
```diff
@@ -104,7 +103,7 @@ DESCONTO HERDADO
 
 ANÚNCIOS
 • Usuários Gratuitos - Nunca enviamos qualquer dado de localização, mas é provável que usem o seu IP (por exemplo) para compreender a sua localização ampla. Isso acontece quando recebe anúncios da sua área.
-• Desabilitamos completamente o código de anúncios, portanto não podem executar ou conhecer dado algum.
+• Usuários Supercharge - Desabilitamos completamente o código de anúncios, portanto não podem executar ou conhecer dado algum.
 
 Envie um e-mail em inglês para team@weatherlineapp.com caso haja qualquer questionamento!
```
Comment (teuscard) [inline pt-BR App Store:108]: Should the user be informed that the email must be written in English?

File: pt-BR App Store [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Por que se limitar a uma fonte quando você pode ter o melhor de cada uma delas? Combinamos os melhores dados de previsão + os melhores dados de chuva + os melhores radares a fim de criar uma super previsão sem pontos fracos que só usa o que há de melhor.
+
+▶ ASSISTENTE DE VIAGEM – PRIMEIRO DA INDÚSTRIA
+
+Nunca mais tenha a localização Atual e a de Casa duplicadas.
+
+▶ 18 TEMAS
+
+• Escolha o seu tema para as aparências clara/escura do iOS
+• 18 temas, de Buraco Nego a Emoji e Funfetti
```
Comment (teuscard) [inline pt-BR App Store:38]: Will theme names be translated in the app?

**Comment by rjonesy** [comment]
Hi @teuscard! I should have marked this repo as deprecated, I thought I alerted everyone that we moved to [google sheets](https://docs.google.com/spreadsheets/d/1Z5Y60IpF-ivpZWaqZwNeawX_V0QO_UYIJ9JexaiVlb8/edit?usp=sharing). I'm really sorry, you must have slipped through the cracks.

But I can transfer your work over there for you. And you bring up great questions that I'm surprised others did not find so thank you. I will address them in the Google sheet shortly.

File: pt-BR App Store (hunk 3 above)
Comment (rjonesy) [reply pt-BR App Store:38]: Ported question to Google Sheets

File: pt-BR App Store (hunk 2 above)
Comment (rjonesy) [reply pt-BR App Store:108]: Nah, we can use google translate, solid solid question though.

File: pt-BR App Store (hunk 1 above)
Comment (rjonesy) [reply pt-BR App Store:17]: I suppose this is irrelevant now with Google Sheets.

**Comment by rjonesy** [comment]
It looks like you've taken great care, I'd love to move these other to the Google Sheet for you, I have a script that can do it. If you'd like to do that, let me know, and submit your PR.

**Comment by teuscard** [comment]
You can move what is needed to the Google Sheet. Therefore, I can translate the remaining file there.

**Comment by rjonesy** [comment]
OK I moved 75% of it. Many of them were already done, so I had to just make a call which one I thought fit better, and obviously I’m not equipped to make that call! So you can replace others if you think you’re are better.

Also please add your email address to the sheet.

-- 
Ryan

**Comment by teuscard** [comment]
How do I access the sheet? I will send you my email address on Twitter.

**Comment by rjonesy** [comment]
Just open it  https://docs.google.com/spreadsheets/d/1Z5Y60IpF-ivpZWaqZwNeawX_V0QO_UYIJ9JexaiVlb8/edit

On May 25, 2020, at 4:26 PM, Mateus Cardoso <notifications@github.com> wrote:

﻿
How do I access the sheet? I will send you my email address on Twitter.

**Pull request merged**